tests/test_gossip.py

```diff
@@ -1964,3 +1964,18 @@ def test_addgossip(node_factory):
 
     with pytest.raises(RpcError, match='Bad signature'):
         l3.rpc.addgossip(badupdate)
+
+
+def test_parms_listforwards(node_factory):
+    """
+    Simple test to ensure that the order of the listforwards
+    is correct as describe in the documentation.
+
+    This test is written by a issue report in the IR channel,
+    it is simple and not useful, but it is good to have to avoid
+    simile errors in the future.
+    """
+    l1, _ = node_factory.line_graph(2)
```

Comment:
Just do node_factory.get_node(options={'allow-deprecated-apis': True}) ?

Reply:
ANd even better would be to add it to test_listforwards rather than having another test.

You could even restart l2 in that test with deprecated-apis enabled, to test?

    l2.stop()
    l2.daemon.opts['allow-deprecated-apis'] = True
    l2.start()

Reply:
Thanks, rusty to point me out this detail. I was thinking that maybe it is better to write the test for the RPC interface in a new file test_rpc? In this way, we can track also the deprecated API and the test could fail in the deprecated period it is over.

What do you think?

In this case, do we need to have some convention to remove the code that it is over?

Reply:
Now should be fixed! Thanks @rustyrussell